Mathlib/Combinatorics/SimpleGraph/Matching.lean

```diff
@@ -368,10 +377,12 @@ for `SimpleGraph` rather than `SimpleGraph.Subgraph`.
 def IsAlternating (G G' : SimpleGraph V) :=
   ∀ ⦃v w w': V⦄, w ≠ w' → G.Adj v w → G.Adj v w' → (G'.Adj v w ↔ ¬ G'.Adj v w')
 
-lemma IsPerfectMatching.symmDiff_spanningCoe_of_isAlternating {M : Subgraph G}
-    (hM : M.IsPerfectMatching) (hG' : G'.IsAlternating M.spanningCoe) (hG'cyc : G'.IsCycles)  :
-    (SimpleGraph.toSubgraph (M.spanningCoe ∆ G')
-      (by rfl)).IsPerfectMatching := by
+lemma IsAlternating.mono {G'' : SimpleGraph V} (halt : G.IsAlternating G') (h : G'' ≤ G) :
+    G''.IsAlternating G' := fun _ _ _ hww' hvw hvw' ↦ halt hww' (h hvw) (h hvw')
+
+lemma IsPerfectMatching.symmDiff_spanningCoe_of_isAlternating (hM : M.IsPerfectMatching)
+    (hG' : G'.IsAlternating M.spanningCoe) (hG'cyc : G'.IsCycles) :
+    (SimpleGraph.toSubgraph (M.spanningCoe ∆ G') (by rfl)).IsPerfectMatching := by
```

Comment:
```suggestion
    ((M.spanningCoe ∆ G').toSubgraph rfl).IsPerfectMatching := by
```
works I think

Reply:
Fixed it to be `(⊤ : Subgraph (M.spanningCoe ∆ G'))`